.github/workflows/sample-apps.yaml

```diff
@@ -13,30 +13,30 @@ jobs:
     name: Publish sample app images
     runs-on: ubuntu-latest
     strategy:
-        matrix:
-          app: [cpu-load-gen, hello-world]
+      matrix:
+        app: [cpu-load-gen, hello-world, outbound-http]
     env:
       IMAGE_NAME: ${{ github.repository }}
 
     steps:
-        - uses: actions/checkout@v4
-
-        - name: Set the release version
-          shell: bash
-          run: |
-            echo "RELEASE_VERSION=$(date +%Y%m%d-%H%M%S)-g$(git rev-parse --short HEAD)" >> $GITHUB_ENV
-
-        - name: Install Spin
-          uses: fermyon/actions/spin/setup@v1
-
-        - name: Install TinyGo
-          uses: acifani/setup-tinygo@v2
-
-        - name: Build and push versioned image
-          uses: fermyon/actions/spin/push@v1
-          with:
-            registry: ${{ env.REGISTRY }}
-            registry_username: ${{ github.actor }}
-            registry_password: ${{ secrets.GITHUB_TOKEN }}
-            registry_reference: "ghcr.io/${{ github.repository_owner }}/spin-operator/${{ matrix.app }}:${{ env.RELEASE_VERSION }}"
-            manifest_file: apps/${{ matrix.app }}/spin.toml
+      - uses: actions/checkout@v4
```

Comment:
Can we keep the indent? 🙈 (I find it way easier to follow yaml when lists follow the same nesting rules as maps)

Reply:
sure thing. Updated!